docs/introduction/nervos-intro.md

```diff
@@ -57,4 +57,4 @@ The CKB economic model focuses on the state. The native token (CK Bytes) represe
 
 Owners utilize their CK Bytes to store state or can lend capacity to others. An implicit storage cost proportional to space and time is created: if an owner utilizes their cell to store state, they will incur an opportunity cost equivalent to what they could have earned by lending out the capacity. This is the CKB's solution to the 'state bloat' problem.
 
-To learn more about the CKB token economics, please refer [token economics paper](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0015-ckb-cryptoeconomics/0015-ckb-cryptoeconomics.md).
\ No newline at end of file
+To learn more about the CKB token economics, please refer [token economics paper](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0015-ckb-cryptoeconomics/0015-ckb-cryptoeconomics.md).
```

Comment:
What is this about?

Reply:
I did not touch this line. Might some white space added somewhere, don't know why.

Reply:
I will revert this line anyway

Reply:
Now I see. It's a newline added by Github editor automatically, so this is good, let's keep it.